Get started guide for RIF Relay Sample dApp SDK (#257)
rsksmart/rif-relay: Readme.md

Each module has instructions for development and usage.

## Getting Started with the RIF Relay Sample dApp SDK

> We've created a sample dApp to showcase how users can submit relayed transactions to the RSK blockchain using the RIF Relay Sample dApp SDK. For details, see [RIF Relaying Services SDK sample dApp](https://github.com/rsksmart/rif-relay-sample-dapp).

Reviewer: ```suggestion
We've created a sample dApp to showcase how users can submit relayed transactions to the Rootstock blockchain using the RIF Relay Sample dApp SDK. For details, see [RIF Relaying Services SDK sample dApp](https://github.com/rsksmart/rif-relay-sample-dapp).
```

We will do the following;

- Run the Rootstock node using JAR and Docker
- Add a network to Metamask
- Setup RIF Relay Contracts and Server
- Configure and run the sample dApp

### Installation Requirements

Ensure your system meets the necessary [installation requirements](https://dev.rootstock.io/guides/rif-relay/installation-requirements/).

### Run the Rootstock Node
The Rootstock node enhances network security, smart contract functionality, privacy, and access to real-time data. It also supports the RSK ecosystem and offers incentives like transaction fees or block rewards. Setup the node on `--regtest mode` using [JAR package](https://dev.rootstock.io/rsk/node/install/operating-systems/java/) or [Docker container](https://dev.rootstock.io/rsk/node/install/operating-systems/).

### Add Network to Metamask
To interact with the Rootstock network, you need to add it to Metamask. Follow the steps in [How to add Metamask to Rootstock](https://dev.rootstock.io/develop/wallet/use/metamask/) to add the Regtest Network since we're using the node on `--regtest mode`.

### Setup RIF Relay Contracts and Server

> The setup of RIF Relay contracts and server is a multi-step process. It involves deploying on-chain components, meeting installation requirements, and using various development tools. For detailed instructions and necessary resources for this setup, see [Setup RIF Relay Contract and Server guide](https://dev.rootstock.io/guides/rif-relay/deployment/). 

Reviewer 2: ```suggestion
The setup of RIF Relay Contracts and Server is a multi-step process. It involves deploying on-chain components, meeting installation requirements, and using various development tools. For detailed instructions and necessary resources for this setup, see [Setup RIF Relay Contract and Server guide](https://dev.rootstock.io/guides/rif-relay/deployment/). 
```

For specific resources, see:
- [RIF Relay Server](https://github.com/rsksmart/rif-relay-server).
- [RIF Relay Contracts](https://github.com/rsksmart/rif-relay-contracts).

### Configure and Run the RIF Relay Sample dApp

> This is a sample dApp to show you how to submit relayed transactions to the Rootstock blockchain using the [RIF Relay SDK](https://github.com/infuy/relaying-services-sdk). Connect to the dApp with MetaMask for signing transactions with the account that owns the Smart Wallets. For more detials, see [RIF Relaying Services SDK sample dApp](https://github.com/rsksmart/rif-relay-sample-dapp).

Reviewer 2: ```suggestion
This is a sample dApp to show you how to submit relayed transactions to the Rootstock blockchain using the [RIF Relay Client](https://github.com/rsksmart/rif-relay-client). Connect to the dApp with MetaMask for signing transactions with the account that owns the Smart Wallets. For more detials, see [RIF Relay Sample dApp](https://github.com/rsksmart/rif-relay-sample-dapp).
```
